heareval/predictions/task_predictions.py

```diff
@@ -904,6 +981,125 @@ def hparams_to_json(hparams):
     return {k: serialize_value(v) for k, v in hparams.items()}
 
 
+def data_splits_from_folds(folds: List[str]) -> List[Dict[str, List[str]]]:
+    """
+    Create data splits by using Leave One Out Cross Validation strategy.
+
+    folds is a list of dataset partitions created during pre-processing. For example,
+    for 5-fold cross val: ["fold00", "fold01", ..., "fold04"]. This function will create
```

Comment:
Is this the final naming convention that we settled upon?

Reply:
yes